tests/serialize/data.js

```diff
@@ -99,7 +99,7 @@ var doNext = function (remaining) {
             return
           }
           doc = $rdf.sym($rdf.uri.join(right, base))
-          if (doc.uri.slice(0, 8) !== 'file:///') {
+          if (doc.uri.slice(0, 7) !== 'file://') {
```

Comment:
I was trying to streamline this and I found out that xmlhttprequest on windows require "file://C:/..." with 2 slashes to retrieve files from the disk. 
https://github.com/driverdan/node-XMLHttpRequest/issues/107
Don't ask me why. Again this apparently fit automafically in the fetcher, the problem is that in this way the uri in the literal referring the code is using the 2 slashes format. This is the case in the main library too, if you feed fetcher.nowOrWhenFetched with the 3 slashes uri it will break.
```
NamedNode {
  termType: 'NamedNode',
  value: 'file:///C:/Users/enrico/test.ttl' }
Uri presented to xhr: file:///C:/Users/enrico/test.ttl
Masked error - status 0 for file:///C:/Users/enrico/test.ttl
xhr content:
{ UNSENT: 0,
  OPENED: 1,
  HEADERS_RECEIVED: 2,
  LOADING: 3,
  DONE: 4,
  readyState: 4,
  onreadystatechange: [Function],
  responseText: 'Error: ENOENT: no such file or directory, open \'C:\\C:\\Users\\enrico\\test.ttl\'\n    at Error (native)',
  responseXML: '',
  status: 0,
  statusText:
   { Error: ENOENT: no such file or directory, open 'C:\C:\Users\enrico\test.ttl'
       at Error (native)
     errno: -4058,
     code: 'ENOENT',
     syscall: 'open',
     path: 'C:\\C:\\Users\\enrico\\test.ttl' },
  withCredentials: false,
  open: [Function],
  setDisableHeaderCheck: [Function],
  setRequestHeader: [Function],
  getResponseHeader: [Function],
  getAllResponseHeaders: [Function],
  getRequestHeader: [Function],
  send: [Function],
  handleError: [Function],
  abort: [Function],
  addEventListener: [Function],
  removeEventListener: [Function],
  dispatchEvent: [Function],
  req: BlankNode { termType: 'BlankNode', id: 1, value: '1' },
  original:
   NamedNode {
     termType: 'NamedNode',
     value: 'file:///C:/Users/enrico/test.ttl' },
  options: { userCallback: [Function] },
  resource:
   NamedNode {
     termType: 'NamedNode',
     value: 'file:///C:/Users/enrico/test.ttl' },
  onerror: [Function],
  timeout: 5000,
  actualProxyURI: 'file:///C:/Users/enrico/test.ttl',
  requestedURI: 'file:///C:/Users/enrico/test.ttl',
  ontimeout: [Function],
  handleResponseDone: true,
  headers: {} }
```